ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# ISSUE
+
+## Reproduction Steps
+
```

Comment:
I have never found better wording then:

* Steps to reproduce
* Actual Results
* Expected Results